kubernetes/resource_kubernetes_daemon_set_v1_test.go

```diff
@@ -685,6 +685,7 @@ func testAccKubernetesDaemonSetV1ConfigWithTemplateMetadata(depName, imageName s
         container {
           image = "%s"
           name  = "containername"
+          command = ["sleep", "infinity"]
```

Comment:
note: the container commands must be added as by default the container just exits and the test is thus failing. This is now added for the majority of the test cases for this resource.